### PR TITLE
Add plain-text version of EULA, include in archives.

### PR DESCRIPTION
**Pull request by mellinoe**
Addresses https://github.com/dotnet/cli/issues/3037.

I've created a text version of eula.rtf. I did this by copy-pasting the contents of eula.rtf from Word 2016 into a text editor, and then fixing up the spacing of the document.

This file, as well as THIRD-PARTY-NOTICES are included in the zip/tar archives we produce from here. They are included in the root of the archive.

@leecow , @joshfree , @richlander

### CODE REVIEW

File: build_projects/dotnet-cli-build/PackageTargets.cs [hunk 1]
```diff
@@ -159,6 +159,14 @@ public static BuildTargetResult CopyCombinedFrameworkSDKHostLayout(BuildTargetCo
             string hostFxrPublishRoot = c.BuildContext.Get<string>("HostFxrPublishRoot");
             Utils.CopyDirectoryRecursively(hostFxrPublishRoot, combinedRoot);
 
+            string licenseSource = Path.Combine(Dirs.RepoRoot, "packaging", "LICENSE.txt");
```
Comment (livarcocc) [inline build_projects/dotnet-cli-build/PackageTargets.cs]: Can we extract this duplication into a helper method?

File: packaging/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+13. LIMITATION ON AND EXCLUSION OF REMEDIES AND DAMAGES. YOU CAN RECOVER FROM MICROSOFT AND ITS SUPPLIERS ONLY DIRECT DAMAGES UP TO U.S. $5.00. YOU CANNOT RECOVER ANY OTHER DAMAGES, INCLUDING CONSEQUENTIAL, LOST PROFITS, SPECIAL, INDIRECT OR INCIDENTAL DAMAGES.
+This limitation applies to
+  * anything related to the software, services, content (including code) on third party Internet sites, or third party programs; and
+  * claims for breach of contract, breach of warranty, guarantee or condition, strict liability, negligence, or other tort to the extent permitted by applicable law.
+It also applies even if Microsoft knew or should have known about the possibility of the damages. The above limitation or exclusion may not apply to you because your country may not allow the exclusion or limitation of incidental, consequential or other damages.
+Please note: As this software is distributed in Quebec, Canada, some of the clauses in this agreement are provided below in French.
+Remarque : Ce logiciel �tant distribu� au Qu�bec, Canada, certaines des clauses dans ce contrat sont fournies ci-dessous en fran�ais.
```
Comment (livarcocc) [inline packaging/LICENSE.txt]: You have some weird characters from here on out. Is this ok?

Comment (mellinoe) [inline packaging/LICENSE.txt]: These are French characters like é, », à, etc. They display for me okay in a text editor, but not when viewed in cmd. Not sure what exactly to do about them. I didn't want to deviate the text from the original version.

File: build_projects/dotnet-cli-build/PackageTargets.cs (hunk 1 above)
Comment (mellinoe) [inline build_projects/dotnet-cli-build/PackageTargets.cs]: ok

File: packaging/LICENSE.txt (hunk 2 above)
Comment (joshfree) [inline packaging/LICENSE.txt]: Keep the original text.

Comment (akoeplinger) [inline packaging/LICENSE.txt]: @mellinoe the encoding of this file is wrong, you need to save it as utf8

**Comment by mellinoe** [comment]
This shouldn't be needed any longer; changes are in core-setup.

**Pull request closed without merging**